### DISCUSSION TITLE
Exact Matching vs Potential Matching

### DISCUSSION THREAD

**Discussion by kodeplex** [moj-analytical-services/splink#1623] [General]
I am trying to cluster potential match on address but have not been successful, does Splink work well with Fuzzy matchicing?

Here is my use case:
**Company Id, Name,                     Address,        State, Postal Code**
10000000, John Medical Services, 2639 Main St, Texas, 65983
10000048, John Medical Services, 2639 Main Street, Texas, 65983
10000056, John Medical Services, 2693 Main Str, Texas, 65983

Is there a way to say that: I set in my config that:
-State, Postal Code should be exact match.
-Address should be: Fuzzy (Levenstein).

And I can get the three records within a same cluster? They're same entity just variation in the address (should be considered potential duplicate, due to variation in the string)

# Specify a data linkage model

```

settings = {
    "link_type": "dedupe_only",
    "blocking_rules_to_generate_predictions": [
        "l.AccountName = r.AccountName",
        "l.PostalCode= r.PostalCode",
        "l.State= r.State",

    ],
    "comparisons": [
        levenshtein_at_thresholds("Address", 2),
        exact_match("State")
    ],
}

```

**Comment by RobinL** [comment]
Your problem seems very similar to the one here:
https://github.com/moj-analytical-services/splink/discussions/1627

[here](https://github.com/moj-analytical-services/splink/discussions/1627#discussioncomment-7255872) is my suggested answer
